Formplayer/userInputs/generate_random_string.py

```diff
@@ -0,0 +1,23 @@
+import random
+import string
+
+""""Genrates random strings that are passed as user inputs across various areasn in CCHQ"""
+
+chars = string.ascii_lowercase + string.digits
+random_string = ''.join(random.choices(chars, k=6))
+N = 10
+random_number = random.randint(100, 19999)
+
+
+def fetch_random_string():
+    return random_string
+
+
+def fetch_phone_number():
+    min_value = pow(10, N - 1)
+    max_value = pow(10, N) - 1
+    return str(random.randint(min_value, max_value))
+
+
+def fetch_random_digit():
+    return str(random_number)
```

Comment:
modify this function with 2 arguements: start and end. and return "random.randint(start, end)" So that it can be used as a generic function.

Reply:
Done!